kterm.c

```diff
@@ -634,6 +662,9 @@ gint main(gint argc, gchar **argv) {
                 i = atoi(optarg);
                 if (i > 0) conf->font_size = (guint) i;
                 break;
+            case 'u':
+                if (optarg[0] == 'B' || optarg[0] == 'I' || optarg[0] == 'U') { conf->cursor_shape = optarg[0]; }
+                break;
```

Comment:
Alphabetical order is a bit lost here but you might also move it after `t` here.